app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = [Person(person["name"], person["age"]) for person in people]
+
+    for index, person in enumerate(people):
+        if "wife" in person.keys() and person["wife"] is not None:
+            wife = Person(person["wife"], 0)
```

Comment:
You can get wife from Person.people[person["wife"]]. No need to create a new obj